packages/expo-modules-autolinking/src/reactNativeConfig/reactNativeConfig.ts

```diff
@@ -77,9 +79,17 @@ export async function findDependencyRootsAsync(
     ...Object.keys(packageJson.dependencies ?? {}),
     ...Object.keys(packageJson.devDependencies ?? {}),
   ];
+  const config = getConfig(projectRoot, {
+    skipSDKVersionRequirement: true,
+  });
```

Comment:
recommend to add `edgeToEdgeEnabled` to gradle.properties and we can read the properties in expo-modules-autolinking. some benefit
1. we don't have to `getConfig` again here (which takes a little time)
2. we don't have to deal with the problem where `expo-modules-autolinking` doesn't have direct dependency to `@expo/config` and `@expo/config-plugins`. (you cannot import them directly)

example of gradle properties reading: https://github.com/expo/expo/blob/da3785482d17e2fd82fa7e0bd9c89efda32d7719/packages/expo-modules-autolinking/src/platforms/android.ts#L124-L137